server/services/client.ts

```diff
@@ -333,13 +333,37 @@ const clientService: (context: StrapiContext) => IClientService = ({ strapi }) =
         default:
           const publishedItems = items.filter(filterOutUnpublished);
           const result = isNil(rootPath) ? items : filterByPath(publishedItems, rootPath).items;
+
+          const defaultCache = new Map<Id, number[]>();
+          const getNestedOrders = (id: Id, cache: Map<Id, number[]> = defaultCache): number[] => {
```

Comment:
```suggestion
          const getNestedOrders = (id: Id, cache: Map<Id, number[]> = defaultCache): Array<number> => {
```

Reply:
To be consistent, I've changed all `number[]` to `Array<number>` 👌